gennav/planners/rrt/informed_rrtstar.py

```diff
@@ -45,16 +44,14 @@ def plan(self, start, goal, env):
 
         Returns:
             [gennav.utils.Trajectory]: The path as a Trajectory
```

Comment:
```suggestion
            gennav.utils.Trajectory, info: The path as a Trajectory along with information dictionary
```

Reply:
Could you make this change for all the planners?

Reply:
Ok